### DISCUSSION TITLE
Looking for help creating an ansible role using this collection

### DISCUSSION THREAD

**Discussion by Tronde** [containers/ansible-podman-collections#551] [General]
Hello folks,  
I'm not sure if this is the right place to look for help with my problem. Please, point me in the correct direction in case this is not the right place.

## What do I have?

I have a working BASH script called [deploy-seafile-pro-with-podman.sh](https://github.com/Tronde/ansible_role_deploy_seafile_with_rootless_podman/blob/main/files/deploy-seafile-pro-with-podman.sh) to deploy a [Seafile Pro Server](https://www.seafile.com/en/product/private_server/) in a rootless Podman environment.

## What would I like to do?

To be able to deploy Seafile instances to rootless Podman environments I would like to create an Ansible role that uses the Containers.Podman collection. This way it would be easy to re-use the role to deploy various instances on the same or other hosts just setting the appropriate input variables for a specific deployment.

I've started creating a role and published it at [Tronde/ansible_role_deploy_seafile_with_rootless_podman](https://github.com/Tronde/ansible_role_deploy_seafile_with_rootless_podman).

## Where do I need help?

While the BASH script [deploy-seafile-pro-with-podman.sh](https://github.com/Tronde/ansible_role_deploy_seafile_with_rootless_podman/blob/main/files/deploy-seafile-pro-with-podman.sh) just works fine the Ansible role doesn't deploy a running and usable Seafile instance and I don't know why.

So I would like to have some second pair of eyes that would take a look at it and maybe spot the issue right away. If you have some spare time to look at it and give me some helpful advice it would be much appreciated.

Kind regards,  
Joerg

**Comment by sshnaidm** [comment]
Please paste your log here or at least the part that fails. We need to know where to look into.

**Comment by Tronde** [comment]
Hi,  
I was able to resolve the issue by using the [Playbook Debugger](https://docs.ansible.com/ansible/2.9/user_guide/playbooks_debugger.html#playbook-debugger). The root cause was a that I was using the wrong variable in some task.

So this discussion could be closed or deleted if possible.

Regards,  
Joerg
